segmentation/cells/UNet3DPlatyCellProbs.model/UNet3DPlatyCellProbs.model.yaml

```diff
@@ -0,0 +1,62 @@
+name: 3D UNet Platynereis Cell Segmentation (Probabilities)
+description: A 3d U-Net trained to predict the cell boundaries in a EM volume of a 6 day old Platynereis.
+cite:
+  - text: Vergara, Hernando M. et al. Whole-body integration of gene expression and single-cell morphology. BioRxiv 2020."
+    doi: https://doi.org/10.1101/2020.02.26.961037
+authors:
+  - Constantin Pape;@bioimage-io
+documentation: ../../README.md
+tags: [unet3d, pytorch, platynereis, cell membrane, segmentation, EM]
+license: MIT
+
+format_version: 0.1.0
+language: python
+framework: pytorch
+
+source: mmpb.segmentation.network.models.UNetAnisotropic
+optional_kwargs:
+  in_channels: 1
+  out_channels: 12
+  initial_features: 22
+  gain: 2
+  scale_factors: [2, 2, 2, 2]
+  final_activation: Sigmoid
+
+test_input: test_input.npz
+test_output: test_output.npz
+covers: [] # todo
```

Comment:
Don't we need covers to properly display this on bioimage.io?

Reply:
it would be nice. I'll add an ilastik screenshot...